### DISCUSSION TITLE
x-collapse.min stays hidden when uncollapsed

### DISCUSSION THREAD

**Discussion by stefanfortuin** [alpinejs/alpine#2925] [7. Bugs (archived)] [answered]
When I add x-collapse.min.{any_height}px and then uncollapse it, it stays hidden and doesn't animate or gets visible.

In the style property display: none; stays there forever.

Normal x-collapse and duration works fine.

```html
<div {{$attributes->class([])}} x-data="{ expanded: false }">
    <div class="whitespace-pre-line prose prose-secondary" x-show="expanded" x-collapse>
        {!! $content!!}
    </div>
    <div class="text-primary cursor-pointer" x-on:click="expanded = !expanded">Lees meer</div>
</div>
```

**Comment by SimoTod** [comment, marked as answer]
Your example works for me
https://codepen.io/SimoTod/pen/XWZjNaK
Make sure you use the right version of Alpine and the plugin as the min modifier is fairly recent

**Comment by stefanfortuin** [comment]
Well... I should have looked better. My Alpine version was indeed out of date.
Thanks
